src/display/editor/tools.js

```diff
@@ -768,8 +768,8 @@ class AnnotationEditorUIManager {
     );
   }
 
-  editAltText(editor) {
-    this.#altTextManager?.editAltText(this, editor);
+  editAltText(editor, isFromKeyboard) {
+    this.#altTextManager?.editAltText(this, editor, isFromKeyboard);
```

Comment:
These changes are no longer necessary, please revert this.
```suggestion
  editAltText(editor) {
    this.#altTextManager?.editAltText(this, editor);
```